android/app/build.gradle

```diff
@@ -186,7 +186,7 @@ android {
         release {
             // Caution! In production, you need to generate your own keystore file.
             // see https://reactnative.dev/docs/signed-apk-android.
-            signingConfig signingConfigs.debug
+            // signingConfig signingConfigs.debug
```

Comment:
Hoezo is deze uitgecommend?

Reply:
Oja kan er uit idd. Dit moet eruit zodat die niet een debug signing key gebruikt. Maar gewoon onze prod